test/framework/src/main/java/org/elasticsearch/bootstrap/BootstrapForTesting.java

```diff
@@ -71,6 +73,21 @@ public class BootstrapForTesting {
 
         // Log ifconfig output before SecurityManager is installed
         IfConfig.logIfNecessary();
+
+        // Fire up entitlements
+        try {
+            TestEntitlementBootstrap.bootstrap(javaTmpDir, maybePath(System.getProperty("tests.config")));
```

Comment:
I can only find a single reference of `tests.config` where this references elasticsearch.yml, though bootstrap expects a dir.
```
mvn -Dtests.gce=true -Dtests.config=/path/to/config/file/elasticsearch.yml clean test
```